templates/default/pom.xml

```diff
@@ -115,8 +121,6 @@
                     <stripVersion>true</stripVersion>
                     <serverStartTimeout>360</serverStartTimeout>
                     <bootstrapProperties>
-                        <default.http.port>${http.port}</default.http.port>
-                        <default.https.port>${https.port}</default.https.port>
                         <app.context.root>${app.name}</app.context.root>
```

Comment:
As per UFO design:  https://ibm.box.com/s/0rnnsg6sh6lv6edq4f6hzkhrakc3sgu1
we should have in the server.xml: 

```xml
 <server description="${project.name}">
 
 
  <httpEndpoint host="*" httpPort="${default.http.port}"
         httpsPort="${default.https.port}" 
          id="defaultHttpEndpoint"/>
 
  <webApplication location="${project.name}.war" contextRoot="/"/>
 
```

So why don't we add the WAR name via:

`<liberty.bootstrap.project.name>${final.name}</liberty.bootstrap.project.name>`

Reply:
So I had to setup the http and https endpoints the way I did because the mvn verify had issues with the values not getting set when using ${default.https.port} from the bootstrap.properties.  Not sure if that matters.  And I don't define the war name because I had to use the dropins directory to load the application... Think this also had to do with some maven issues.. I have to think a bit and remember why I did it this way. But I think it was to allow the native mvn liberty:dev and mvn verify to run natively...

Reply:
I think since I'm using the implied Docker file for MST I have am forced to use dropins. So no application is being defined.

Reply:
Good point..well, let's checkin a Dockerfile (in lieu of getting it from the stack).